test/cli/test_cli.ml

```diff
@@ -48,6 +48,9 @@ let main ocamlformat =
   test ~name:"fmterr_stdin_bad_kind" ~stdin:intf_file ~args:"--impl -" () ;
   test ~name:"fmterr_file_and_name_bad_kind" ~args:"--name foo.ml sample/a.mli" () ;
   test ~name:"fmterr_stdin_and_name_bad_kind" ~stdin:intf_file ~args:"--name foo.ml -" () ;
+  test ~name:"check_formatted" ~args:"--check sample/a.ml" () ;
+  test ~name:"check_not_formatted" ~args:"--check sample/b.ml" () ;
```

Comment:
These 2 checks are finally not necessary but they feel useful. Should I keep them?
(I added them to check if `Conf.check` is useful when combined with `--max-iters=1`).